controller/backup_controller.go

```diff
@@ -499,6 +499,14 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 		return nil
 	}
 
+	defer func() {
+		if backup.Spec.CleanupSnapshot {
```

Comment:
Can we introduce a global setting for the snapshot cleanup?

Reply:
Introduced a global setting and simplified the flow. Thanks.